src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1557,7 +1558,7 @@ namespace winrt::TerminalApp::implementation
         }
 
         // Let's assume the user has bound the dead key "^" to a sendInput command that sends "b".
-        // If the user presses the two keys "^a" it'll produce "bâ", despite us marking the key event as handled.
+        // If the user presses the two keys "^a" it'll produce "bÃ¢", despite us marking the key event as handled.
```

Comment:
```suggestion
        // If the user presses the two keys "^a" it'll produce "bâ", despite us marking the key event as handled.
```

I don't know what happened, but the encoding on this file got big broken